test/integration/che/che.go

```diff
@@ -90,13 +90,13 @@ func (c *CheRunner) theStacksShouldNotBeEmpty() error {
 	return nil
 }
 
-func (c *CheRunner) startingWorkspaceWithStackSucceeds(stackName string) error {
+func (c *CheRunner) startingWorkspaceWithStackSucceeds(stackName string, maxTime string) error {
```

Comment:
suggestion s/maxTime/timeoutValue/

Reply:
>     * timeout string, maxDuration time.Duration

+1. Sounds good to me.

Reply:
@agajdosi Unless you want to make the variable names consistent, I am ok to merge as it is. Mostly because I can still accept the current names. Let me know.

Reply:
@LalatenduMohanty I have kept it consistent around Che test - using the timeout and maxDuration in one func. Rebased now, we can merge.